.github/dependabot.yml

```diff
@@ -5,38 +5,42 @@
 # these updates when the pull request(s) in the appropriate skeleton are merged
 # and Lineage processes these changes.
 
-version: 2
 updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
+  - directory: /
     ignore:
       # Managed by cisagov/skeleton-generic
       - dependency-name: actions/cache
       - dependency-name: actions/checkout
       - dependency-name: actions/setup-go
       - dependency-name: actions/setup-python
+      - dependency-name: crazy-max/ghaction-dump-context
+      - dependency-name: crazy-max/ghaction-github-labeler
+      - dependency-name: crazy-max/ghaction-github-status
       - dependency-name: hashicorp/setup-terraform
       - dependency-name: mxschmitt/action-tmate
+      - dependency-name: step-security/harden-runner
+    package-ecosystem: github-actions
+    schedule:
+      interval: weekly
 
-  - package-ecosystem: "pip"
-    directory: "/"
+  - directory: /
+    package-ecosystem: pip
     schedule:
-      interval: "weekly"
+      interval: weekly
 
-  - package-ecosystem: "terraform"
-    directory: "/"
+  - directory: /
+    package-ecosystem: terraform
     schedule:
-      interval: "weekly"
+      interval: weekly
     # # Managed by cisagov/skeleton-tf-module
     # ignore:
-    #   - dependency-name: "hashicorp/aws"
+    #   - dependency-name: hashicorp/aws
 
-  - package-ecosystem: "terraform"
-    directory: "/examples/basic_usage"
+  - package-ecosystem: terraform
```

Comment:
Please adjust this configuration like I did in https://github.com/cisagov/skeleton-packer/pull/264/commits/58c56ad65bbf411c808ffe3b29514af3ead2a578 to have the entries correctly sorted and formatted.

Reply:
Please see commit 6449792.

Reply:
@jsf9k Please also adjust the attribution comment so it falls under the `ignore` block.

Reply:
I cannot do a suggestion because of how GitHub tries to format this but here is what I mean:

```yaml
    # ignore:
    #   # Managed by cisagov/skeleton-tf-module
    #   - dependency-name: hashicorp/aws
```

Reply:
Please see commit b0011b2.